Tests/Api/ApiLiveTradingTests.cs

```diff
@@ -0,0 +1,346 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Web;
+using NUnit.Framework;
+using QuantConnect.Api;
+using QuantConnect.Brokerages;
+using QuantConnect.Configuration;
+
+namespace QuantConnect.Tests.API
+{
+    [TestFixture, Ignore("These tests require configured and active accounts to Interactive Brokers, FXCM, Oanda, and Tradier " +
+         "as well as your Organization ID and available live nodes")]
+    class ApiLiveTradingTests
```

Comment:
Should be `public` I think? Not sure it will be found by the test framework if not

Reply:
Got it! It seems to work anyway but might as well clarify that it is public!